migrations/2024-02-12-135546_add_fingerprint_id_in_payment_attempt/down.sql

```diff
@@ -0,0 +1,2 @@
+-- This file should undo anything in `up.sql`
+ALTER TABLE payment_attempt ADD COLUMN IF NOT EXISTS fingerprint_id VARCHAR(64);
```

Comment:
`ADD COLUMN`, ??

Reply:
Is there any problem in doing so? We are adding a new column in attempt table to store the fingerprint_id.